starter_code/public/stylesheets/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  padding: 0;
+  margin: 0;
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif; }
+
+a {
+  color: #00b7ff; }
+
+.homepage-container {
+  background-image: url(/images/cinema.jpg);
+  background-size: 100%;
```

Comment:
I think this background-size: 100% is not going to look too good on smaller screens, that is if you would ever have a div underneath .homepage-container.

I would personally go for background-size: 100% 100% or background-size: cover